yb-voyager/src/srcdb/source.go

```diff
@@ -37,6 +37,7 @@ type Source struct {
 	TableList        string
 	ExcludeTableList string
 	UseOrafce        bool
+	CommentsOnObjects bool
```

Comment:
don't you GO extensions installed on your dev-machine? It would automatically correct the indentation and spaces on this line to align with the above ones

Reply:
I do have but don't know why it didn't work sometimes. Will try to install it again.

Reply:
ok